examples/ConfigStoreDemo/Program.cs

```diff
@@ -24,9 +26,13 @@ public static IWebHost BuildWebHost(string[] args)
                     // 3. Set up the provider to listen for changes to the background color key-value in Azure App Configuration
 
                     var settings = config.AddJsonFile("appsettings.json").Build();
+
+                    IConfigurationSection endpointsSection = settings.GetSection("endpoints");
+                    var endpoints = endpointsSection.GetChildren().AsEnumerable().Select(endpoint => new Uri(endpoint.Value));
```

Comment:
Can you use an explicit type instead of `var`. Also we use `var` above, we can change that as well. 

`var` is useful when the type of the variable is clear from the assignment like in `var dict = new Dictionary<string, string>();`